src/main/java/decodes/tsdb/TimeSeriesDb.java

```diff
@@ -593,9 +618,94 @@ public TimeSeriesDb()
 
         cpCompDepends_col1 = isHdb() ? "TS_ID" : "SITE_DATATYPE_ID";
         getLogDateFormat().setTimeZone(TimeZone.getTimeZone("UTC"));
+        setupCaches();
+    }
+
+    private void setupCaches()
```

Comment:
I don't understand the difference in this setupCaches() and the one in SqlDatabaseIO.

Reply:
Well, there really isn't actually, we could make this a static function that takes a DatabaseConnectionOwner and CacheMap?

SqlDatabaseIO vs TimeSeriesDb is one of the unfortunate parts of OpenDCS history - it'll slowly get phased out, but changes like this, even though some work is duplicated are I think necessary to make that happen. E.g. this adds a little complexity but overall simplifies some operations *and* future changes.